roles/docker/defaults/main.yml

```diff
@@ -7,3 +7,8 @@ docker_lvm_data_volume_size: 40%FREE
 docker_lvm_data_volume_size_min: 2G
 
 docker_lvm_auto_extend_pool: yes
+
+docker_gc:
```

Comment:
Because there are two exclusion files, I was thinking of adding in another key in here that would relate to that other file. 
So, the var could be this:
```
docker_gc:
  exclude:
    - list items
  exclude_containers:
    - list items
```
If this doesn't make any sense, I can change it to 
```
docker_c_exclude_containers:
  - list items
```
Let me know if there is a preference.